2018-12-04 08:20:29,039 - detector_disgnosis_detail.py - [line:47] - ERROR - when reaching detector_all, invalid literal for int() with base 10: '88.8'
2018-12-04 11:53:01,407 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, send_oracle_data() missing 1 required positional argument: 'sql'
2018-12-04 11:53:18,001 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, send_oracle_data() missing 1 required positional argument: 'sql'
2018-12-04 11:55:34,280 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, send_oracle_data() missing 1 required positional argument: 'sql'
2018-12-04 11:55:52,207 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, send_oracle_data() missing 1 required positional argument: 'sql'
2018-12-04 11:56:13,566 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, send_oracle_data() missing 1 required positional argument: 'sql'
2018-12-04 11:58:46,619 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, send_oracle_data() missing 1 required positional argument: 'sql'
2018-12-04 11:58:50,622 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, send_oracle_data() missing 1 required positional argument: 'sql'
2018-12-04 12:01:38,340 - maintain_list.py - [line:72] - ERROR - when reaching detector_all, 'NoneType' object is not iterable
2018-12-04 12:02:50,928 - maintain_list.py - [line:72] - ERROR - when insert new maintain list, 'NoneType' object is not iterable
2018-12-04 12:03:16,011 - maintain_list.py - [line:72] - ERROR - when insert new maintain list, 'NoneType' object is not iterable
2018-12-04 12:06:21,545 - maintain_list.py - [line:72] - ERROR - when insert new maintain list, 'NoneType' object is not iterable
2018-12-04 13:31:23,729 - maintain_list.py - [line:80] - ERROR - when insert new maintain list, 'NoneType' object is not subscriptable
2018-12-04 13:31:32,359 - maintain_list.py - [line:80] - ERROR - when insert new maintain list, 'fstr_name'
2018-12-04 13:31:47,608 - maintain_list.py - [line:81] - ERROR - when insert new maintain list, 'fstr_name'
2018-12-04 13:39:55,677 - maintain_list.py - [line:90] - ERROR - when insert new maintain list, name 'maintain' is not defined
2018-12-04 13:40:47,219 - maintain_list.py - [line:91] - ERROR - when insert new maintain list, unsupported operand type(s) for +: 'int' and 'str'
2018-12-04 13:43:28,399 - maintain_list.py - [line:93] - ERROR - when insert new maintain list, unsupported operand type(s) for +: 'int' and 'str'
